github/repos.go

```diff
@@ -849,6 +850,11 @@ type AllowForcePushes struct {
 	Enabled bool `json:"enabled"`
 }
 
+// Requires all comments on the pull request to be resolved before it can be merged to a protected branch
```

Comment:
```suggestion
// RequiredConversationResolution, if enabled, requires all comments on the pull request to be resolved before it can be merged to a protected branch.
```